src/ConfigWidgets/SymbolsDialogTest.cpp

```diff
@@ -39,67 +37,73 @@ class MockPersistentStorageManager : public orbit_symbol_paths::PersistentStorag
               LoadModuleSymbolFileMappings, (), (override));
 };
 
-TEST(SymbolsDialog, ConstructEmpty) {
-  MockPersistentStorageManager mock_storage_manager;
-  EXPECT_CALL(mock_storage_manager, LoadPaths())
-      .WillOnce(testing::Return(std::vector<std::filesystem::path>{}));
-  EXPECT_CALL(mock_storage_manager, SavePaths)
-      .WillOnce([](absl::Span<const std::filesystem::path> paths) { EXPECT_TRUE(paths.empty()); });
+class SymbolsDialogTest : public ::testing::Test {
+ protected:
+  void SetLoadPaths(std::vector<std::filesystem::path> load_paths) {
+    EXPECT_CALL(mock_storage_manager_, LoadPaths).WillOnce(testing::Return(std::move(load_paths)));
+  }
+  void SetExpectSavePaths(std::vector<std::filesystem::path> save_paths) {
+    EXPECT_CALL(mock_storage_manager_, SavePaths)
+        .WillOnce(
+            [save_paths = std::move(save_paths)](absl::Span<const std::filesystem::path> paths) {
+              EXPECT_EQ(paths, save_paths);
```

Comment:
@beckerhe I still used EXPECT_EQ here, because it works with absl::Span (paths) and std::vector (save_paths). testing::ElementsAre does not work (I am not sure why)

Reply:
Fair enough. Thx for trying.